2024-08-06 19:54:38,048 INFO    StreamThr :2144364 [internal.py:wandb_internal():86] W&B internal server running at pid: 2144364, started at: 2024-08-06 19:54:38.048384
2024-08-06 19:54:38,049 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: status
2024-08-06 19:54:38,050 INFO    WriterThread:2144364 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/run-3pl4ow3k.wandb
2024-08-06 19:54:38,050 DEBUG   SenderThread:2144364 [sender.py:send():382] send: header
2024-08-06 19:54:38,050 DEBUG   SenderThread:2144364 [sender.py:send():382] send: run
2024-08-06 19:54:38,210 INFO    SenderThread:2144364 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files
2024-08-06 19:54:38,210 INFO    SenderThread:2144364 [sender.py:_start_run_threads():1136] run started: 3pl4ow3k with start time 1722945278.048164
2024-08-06 19:54:38,214 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: check_version
2024-08-06 19:54:38,214 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: check_version
2024-08-06 19:54:38,306 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: run_start
2024-08-06 19:54:38,315 DEBUG   HandlerThread:2144364 [system_info.py:__init__():32] System info init
2024-08-06 19:54:38,315 DEBUG   HandlerThread:2144364 [system_info.py:__init__():47] System info init done
2024-08-06 19:54:38,315 INFO    HandlerThread:2144364 [system_monitor.py:start():194] Starting system monitor
2024-08-06 19:54:38,315 INFO    SystemMonitor:2144364 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-06 19:54:38,315 INFO    HandlerThread:2144364 [system_monitor.py:probe():214] Collecting system info
2024-08-06 19:54:38,316 INFO    SystemMonitor:2144364 [interfaces.py:start():190] Started cpu monitoring
2024-08-06 19:54:38,316 INFO    SystemMonitor:2144364 [interfaces.py:start():190] Started disk monitoring
2024-08-06 19:54:38,316 INFO    SystemMonitor:2144364 [interfaces.py:start():190] Started gpu monitoring
2024-08-06 19:54:38,317 INFO    SystemMonitor:2144364 [interfaces.py:start():190] Started memory monitoring
2024-08-06 19:54:38,317 INFO    SystemMonitor:2144364 [interfaces.py:start():190] Started network monitoring
2024-08-06 19:54:38,323 DEBUG   HandlerThread:2144364 [system_info.py:probe():196] Probing system
2024-08-06 19:54:38,324 DEBUG   HandlerThread:2144364 [system_info.py:_probe_git():181] Probing git
2024-08-06 19:54:38,326 DEBUG   HandlerThread:2144364 [system_info.py:_probe_git():189] Probing git done
2024-08-06 19:54:38,327 DEBUG   HandlerThread:2144364 [system_info.py:probe():244] Probing system done
2024-08-06 19:54:38,327 DEBUG   HandlerThread:2144364 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-41-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-06T11:54:38.323606', 'startedAt': '2024-08-06T11:54:38.043382', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/qwen2-1.5b', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:395000]', '--output_dir', '/home/rwkv/JL/out_model/Gemma-meta-2btest', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '2', '--gradient_accumulation_steps', '16', '--save_strategy', 'steps', '--save_steps', '1000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': None}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 2437.93, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 3352.547, 'min': 800.0, 'max': 5500.0}, {'current': 1159.859, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 1214.01, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5500.083, 'min': 800.0, 'max': 5500.0}, {'current': 5507.845, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 5500.0, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 5501.878, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 5425.828, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1099.99, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 443.3738212585449}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.5532341003418}}
2024-08-06 19:54:38,327 INFO    HandlerThread:2144364 [system_monitor.py:probe():224] Finished collecting system info
2024-08-06 19:54:38,327 INFO    HandlerThread:2144364 [system_monitor.py:probe():227] Publishing system info
2024-08-06 19:54:38,327 DEBUG   HandlerThread:2144364 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-06 19:54:38,327 DEBUG   HandlerThread:2144364 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-06 19:54:38,327 DEBUG   HandlerThread:2144364 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-06 19:54:39,211 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/conda-environment.yaml
2024-08-06 19:54:39,211 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/requirements.txt
2024-08-06 19:54:40,020 DEBUG   HandlerThread:2144364 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-06 19:54:40,020 DEBUG   HandlerThread:2144364 [system_info.py:_save_code():90] Saving code
2024-08-06 19:54:40,021 DEBUG   HandlerThread:2144364 [system_info.py:_save_code():111] Saving code done
2024-08-06 19:54:40,021 DEBUG   HandlerThread:2144364 [system_info.py:_save_patches():128] Saving git patches
2024-08-06 19:54:40,024 ERROR   HandlerThread:2144364 [system_info.py:_save_patches():169] Error generating diff: Reference at 'refs/remotes/origin/main' does not exist
2024-08-06 19:54:40,024 DEBUG   HandlerThread:2144364 [system_info.py:_save_patches():170] Saving git patches done
2024-08-06 19:54:40,024 INFO    HandlerThread:2144364 [system_monitor.py:probe():229] Finished publishing system info
2024-08-06 19:54:40,025 DEBUG   SenderThread:2144364 [sender.py:send():382] send: files
2024-08-06 19:54:40,025 INFO    SenderThread:2144364 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-06 19:54:40,025 INFO    SenderThread:2144364 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-06 19:54:40,027 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 19:54:40,027 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: stop_status
2024-08-06 19:54:40,028 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 19:54:40,210 DEBUG   SenderThread:2144364 [sender.py:send():382] send: telemetry
2024-08-06 19:54:40,210 DEBUG   SenderThread:2144364 [sender.py:send():382] send: config
2024-08-06 19:54:40,211 DEBUG   SenderThread:2144364 [sender.py:send():382] send: metric
2024-08-06 19:54:40,211 DEBUG   SenderThread:2144364 [sender.py:send():382] send: telemetry
2024-08-06 19:54:40,211 DEBUG   SenderThread:2144364 [sender.py:send():382] send: metric
2024-08-06 19:54:40,211 WARNING SenderThread:2144364 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-06 19:54:40,211 DEBUG   SenderThread:2144364 [sender.py:send():382] send: telemetry
2024-08-06 19:54:40,211 DEBUG   SenderThread:2144364 [sender.py:send():382] send: telemetry
2024-08-06 19:54:40,211 DEBUG   SenderThread:2144364 [sender.py:send():382] send: config
2024-08-06 19:54:40,212 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/conda-environment.yaml
2024-08-06 19:54:40,212 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/code/llama/gmm.py
2024-08-06 19:54:40,212 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/output.log
2024-08-06 19:54:40,212 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/wandb-metadata.json
2024-08-06 19:54:40,212 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/code/llama
2024-08-06 19:54:40,212 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/code
2024-08-06 19:54:40,287 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115440Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a53b276bc2ceb75e1a768da55f4c1263437f34f1406f44325e1929df382a6ccf: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115440Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a53b276bc2ceb75e1a768da55f4c1263437f34f1406f44325e1929df382a6ccf
2024-08-06 19:54:40,287 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14101', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:40,287 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9219A93085D29</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:41,523 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115440Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a53b276bc2ceb75e1a768da55f4c1263437f34f1406f44325e1929df382a6ccf: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115440Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a53b276bc2ceb75e1a768da55f4c1263437f34f1406f44325e1929df382a6ccf
2024-08-06 19:54:41,523 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14101', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:41,523 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9219ADE8289BD</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:41,817 DEBUG   SenderThread:2144364 [sender.py:send():382] send: exit
2024-08-06 19:54:41,817 INFO    SenderThread:2144364 [sender.py:send_exit():589] handling exit code: 1
2024-08-06 19:54:41,817 INFO    SenderThread:2144364 [sender.py:send_exit():591] handling runtime: 3
2024-08-06 19:54:41,817 INFO    SenderThread:2144364 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:54:41,817 INFO    SenderThread:2144364 [sender.py:send_exit():597] send defer
2024-08-06 19:54:41,817 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:41,817 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-06 19:54:41,818 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:41,818 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-06 19:54:41,818 INFO    SenderThread:2144364 [sender.py:transition_state():617] send defer: 1
2024-08-06 19:54:41,818 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:41,818 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-06 19:54:41,818 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:41,818 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-06 19:54:41,818 INFO    SenderThread:2144364 [sender.py:transition_state():617] send defer: 2
2024-08-06 19:54:41,818 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:41,818 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-06 19:54:41,818 INFO    HandlerThread:2144364 [system_monitor.py:finish():203] Stopping system monitor
2024-08-06 19:54:41,818 INFO    HandlerThread:2144364 [interfaces.py:finish():202] Joined cpu monitor
2024-08-06 19:54:41,818 INFO    HandlerThread:2144364 [interfaces.py:finish():202] Joined disk monitor
2024-08-06 19:54:41,818 DEBUG   SystemMonitor:2144364 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-06 19:54:41,818 DEBUG   SystemMonitor:2144364 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-06 19:54:41,818 DEBUG   SystemMonitor:2144364 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-06 19:54:41,871 INFO    HandlerThread:2144364 [interfaces.py:finish():202] Joined gpu monitor
2024-08-06 19:54:41,871 INFO    HandlerThread:2144364 [interfaces.py:finish():202] Joined memory monitor
2024-08-06 19:54:41,871 INFO    HandlerThread:2144364 [interfaces.py:finish():202] Joined network monitor
2024-08-06 19:54:41,871 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:41,871 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-06 19:54:41,871 INFO    SenderThread:2144364 [sender.py:transition_state():617] send defer: 3
2024-08-06 19:54:41,871 DEBUG   SenderThread:2144364 [sender.py:send():382] send: stats
2024-08-06 19:54:41,871 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:41,871 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-06 19:54:41,872 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:41,872 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-06 19:54:41,872 INFO    SenderThread:2144364 [sender.py:transition_state():617] send defer: 4
2024-08-06 19:54:41,872 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:41,872 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-06 19:54:41,872 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:41,872 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-06 19:54:41,872 INFO    SenderThread:2144364 [sender.py:transition_state():617] send defer: 5
2024-08-06 19:54:41,872 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:41,872 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-06 19:54:41,872 DEBUG   SenderThread:2144364 [sender.py:send():382] send: summary
2024-08-06 19:54:41,872 INFO    SenderThread:2144364 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:54:41,872 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:41,872 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-06 19:54:41,872 INFO    SenderThread:2144364 [sender.py:transition_state():617] send defer: 6
2024-08-06 19:54:41,873 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:41,873 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-06 19:54:41,873 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:41,873 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-06 19:54:41,874 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:54:41,953 INFO    SenderThread:2144364 [sender.py:transition_state():617] send defer: 7
2024-08-06 19:54:41,953 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:41,953 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-06 19:54:41,954 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:41,954 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-06 19:54:42,215 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/config.yaml
2024-08-06 19:54:42,225 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/wandb-summary.json
2024-08-06 19:54:42,817 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 19:54:43,213 INFO    Thread-12 :2144364 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/output.log
2024-08-06 19:54:43,728 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115440Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a53b276bc2ceb75e1a768da55f4c1263437f34f1406f44325e1929df382a6ccf: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115440Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a53b276bc2ceb75e1a768da55f4c1263437f34f1406f44325e1929df382a6ccf
2024-08-06 19:54:43,728 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14101', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:43,728 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9219B61EF3C34</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:43,728 INFO    wandb-upload_1:2144364 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115440Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a53b276bc2ceb75e1a768da55f4c1263437f34f1406f44325e1929df382a6ccf

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 19:54:44,227 INFO    SenderThread:2144364 [sender.py:transition_state():617] send defer: 8
2024-08-06 19:54:44,227 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 19:54:44,228 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:44,228 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-06 19:54:44,228 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:44,228 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-06 19:54:44,228 INFO    SenderThread:2144364 [job_builder.py:build():298] Attempting to build job artifact
2024-08-06 19:54:44,228 INFO    SenderThread:2144364 [job_builder.py:_get_source_type():439] no source found
2024-08-06 19:54:44,228 INFO    SenderThread:2144364 [sender.py:transition_state():617] send defer: 9
2024-08-06 19:54:44,228 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:44,229 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-06 19:54:44,229 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:44,229 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 9
2024-08-06 19:54:44,229 INFO    SenderThread:2144364 [dir_watcher.py:finish():358] shutting down directory watcher
2024-08-06 19:54:45,215 INFO    SenderThread:2144364 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/output.log
2024-08-06 19:54:45,216 INFO    SenderThread:2144364 [dir_watcher.py:finish():388] scan: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files
2024-08-06 19:54:45,216 INFO    SenderThread:2144364 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/wandb-summary.json wandb-summary.json
2024-08-06 19:54:45,216 INFO    SenderThread:2144364 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/requirements.txt requirements.txt
2024-08-06 19:54:45,216 INFO    SenderThread:2144364 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/output.log output.log
2024-08-06 19:54:45,217 INFO    SenderThread:2144364 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/config.yaml config.yaml
2024-08-06 19:54:45,217 INFO    SenderThread:2144364 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/wandb-metadata.json wandb-metadata.json
2024-08-06 19:54:45,217 INFO    SenderThread:2144364 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/conda-environment.yaml conda-environment.yaml
2024-08-06 19:54:45,221 INFO    SenderThread:2144364 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/code/llama/gmm.py code/llama/gmm.py
2024-08-06 19:54:45,222 INFO    SenderThread:2144364 [sender.py:transition_state():617] send defer: 10
2024-08-06 19:54:45,229 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:54:45,229 INFO    HandlerThread:2144364 [handler.py:handle_request_defer():172] handle defer: 10
2024-08-06 19:54:45,237 DEBUG   SenderThread:2144364 [sender.py:send_request():409] send_request: defer
2024-08-06 19:54:45,237 INFO    SenderThread:2144364 [sender.py:send_request_defer():613] handle sender defer: 10
2024-08-06 19:54:45,238 INFO    SenderThread:2144364 [file_pusher.py:finish():175] shutting down file pusher
2024-08-06 19:54:45,328 ERROR   wandb-upload_3:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d688368325c45d6b09a95f8bf2f3c6fbe02e28e2b4ea8da80b385cdc12e567de: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d688368325c45d6b09a95f8bf2f3c6fbe02e28e2b4ea8da80b385cdc12e567de
2024-08-06 19:54:45,328 ERROR   wandb-upload_3:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:45,328 ERROR   wandb-upload_3:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E9219BC17EA138</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:45,369 INFO    wandb-upload_2:2144364 [upload_job.py:push():131] Uploaded file /home/rwkv/JL/GMM/llama/wandb/run-20240806_195438-3pl4ow3k/files/wandb-summary.json
2024-08-06 19:54:45,403 ERROR   wandb-upload_5:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0cc41c7e124f12133c7a966d6499ebd4806c976308994ee1d82d7c7e72576691: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0cc41c7e124f12133c7a966d6499ebd4806c976308994ee1d82d7c7e72576691
2024-08-06 19:54:45,403 ERROR   wandb-upload_5:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11161', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:45,403 ERROR   wandb-upload_5:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E9219BC689D16E</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:45,427 ERROR   wandb-upload_4:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0acb7dd9321ebf484fd13f27b81792c15310b9ab08f140ca453bc3acf0c22079: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0acb7dd9321ebf484fd13f27b81792c15310b9ab08f140ca453bc3acf0c22079
2024-08-06 19:54:45,427 ERROR   wandb-upload_4:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '4480', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:45,427 ERROR   wandb-upload_4:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/output.log</Resource><Region>wandb-local</Region><RequestId>17E9219BC7FF312A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:45,428 ERROR   wandb-upload_6:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=56c93ed58e4e9c2435aa070b1b80b539109d536946513227f257c4874d63bb28: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=56c93ed58e4e9c2435aa070b1b80b539109d536946513227f257c4874d63bb28
2024-08-06 19:54:45,429 ERROR   wandb-upload_6:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:45,429 ERROR   wandb-upload_6:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E9219BC7F7089B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:46,382 ERROR   wandb-upload_3:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d688368325c45d6b09a95f8bf2f3c6fbe02e28e2b4ea8da80b385cdc12e567de: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d688368325c45d6b09a95f8bf2f3c6fbe02e28e2b4ea8da80b385cdc12e567de
2024-08-06 19:54:46,382 ERROR   wandb-upload_3:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:46,382 ERROR   wandb-upload_3:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E9219C00B64FF9</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:46,539 ERROR   wandb-upload_5:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0cc41c7e124f12133c7a966d6499ebd4806c976308994ee1d82d7c7e72576691: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0cc41c7e124f12133c7a966d6499ebd4806c976308994ee1d82d7c7e72576691
2024-08-06 19:54:46,539 ERROR   wandb-upload_5:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11161', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:46,539 ERROR   wandb-upload_5:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E9219C09C0F1DA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:46,607 ERROR   wandb-upload_6:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=56c93ed58e4e9c2435aa070b1b80b539109d536946513227f257c4874d63bb28: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=56c93ed58e4e9c2435aa070b1b80b539109d536946513227f257c4874d63bb28
2024-08-06 19:54:46,607 ERROR   wandb-upload_6:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:46,607 ERROR   wandb-upload_6:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E9219C0DA74A11</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:47,085 ERROR   wandb-upload_4:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0acb7dd9321ebf484fd13f27b81792c15310b9ab08f140ca453bc3acf0c22079: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0acb7dd9321ebf484fd13f27b81792c15310b9ab08f140ca453bc3acf0c22079
2024-08-06 19:54:47,085 ERROR   wandb-upload_4:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '4480', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:47,085 ERROR   wandb-upload_4:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/output.log</Resource><Region>wandb-local</Region><RequestId>17E9219C14ACD6C7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:47,819 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:54:48,700 ERROR   wandb-upload_5:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0cc41c7e124f12133c7a966d6499ebd4806c976308994ee1d82d7c7e72576691: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0cc41c7e124f12133c7a966d6499ebd4806c976308994ee1d82d7c7e72576691
2024-08-06 19:54:48,700 ERROR   wandb-upload_5:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11161', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:48,700 ERROR   wandb-upload_5:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E9219C8A146618</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:48,734 ERROR   wandb-upload_6:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=56c93ed58e4e9c2435aa070b1b80b539109d536946513227f257c4874d63bb28: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=56c93ed58e4e9c2435aa070b1b80b539109d536946513227f257c4874d63bb28
2024-08-06 19:54:48,734 ERROR   wandb-upload_6:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:48,734 ERROR   wandb-upload_6:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E9219C8C97760A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:48,746 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115440Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a53b276bc2ceb75e1a768da55f4c1263437f34f1406f44325e1929df382a6ccf: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115440Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=a53b276bc2ceb75e1a768da55f4c1263437f34f1406f44325e1929df382a6ccf
2024-08-06 19:54:48,746 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14101', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:48,746 ERROR   wandb-upload_1:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9219C8D2F91CA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:48,872 ERROR   wandb-upload_3:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d688368325c45d6b09a95f8bf2f3c6fbe02e28e2b4ea8da80b385cdc12e567de: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=d688368325c45d6b09a95f8bf2f3c6fbe02e28e2b4ea8da80b385cdc12e567de
2024-08-06 19:54:48,872 ERROR   wandb-upload_3:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:48,872 ERROR   wandb-upload_3:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E9219C9534AA2C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:54:49,239 DEBUG   HandlerThread:2144364 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:54:49,485 ERROR   wandb-upload_4:2144364 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0acb7dd9321ebf484fd13f27b81792c15310b9ab08f140ca453bc3acf0c22079: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/3pl4ow3k/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115445Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0acb7dd9321ebf484fd13f27b81792c15310b9ab08f140ca453bc3acf0c22079
2024-08-06 19:54:49,485 ERROR   wandb-upload_4:2144364 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '4480', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:54:49,485 ERROR   wandb-upload_4:2144364 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/3pl4ow3k/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/3pl4ow3k/output.log</Resource><Region>wandb-local</Region><RequestId>17E9219CB9B72080</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
